ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPoolLondonTest.java

```diff
@@ -193,17 +194,92 @@ public void shouldAcceptBaseFeeFloorGasPriceFrontierTransactionsWhenMining() {
     assertLocalTransactionValid(frontierTransaction);
   }
 
+  @Test
+  public void shouldRejectRemote1559TxsWhenMaxFeePerGasBelowMinGasPrice() {
+    final Wei genesisBaseFee = Wei.of(100L);
+    final Wei minGasPrice = Wei.of(200L);
+    final Wei lastBlockBaseFee = minGasPrice.add(50L);
+    final Wei txMaxFeePerGas = minGasPrice.subtract(1L);
+
+    assertThat(
+            add1559TxAndGetPendingTxsCount(
+                genesisBaseFee, minGasPrice, lastBlockBaseFee, txMaxFeePerGas, false))
+        .isEqualTo(0);
+  }
+
+  @Test
+  public void shouldAcceptRemote1559TxsWhenMaxFeePerGasAboveMinGasPrice() {
```

Comment:
txMaxFeePerGas = minGasPrice
it's not above but equal - should rename test?

Reply:
renamed